pcomfortcloud/__main__.py

```diff
@@ -50,7 +50,7 @@ def main():
     parser.add_argument(
         '-t', '--token',
         help='File to store token in',
-        default='token.json')
+        default='~/.pcomfortcloud-oauth-token')
```

Comment:
A thought. You probably should be using. The config path. Instead of ~ here. 

https://unix.stackexchange.com/questions/312988/understanding-home-configuration-file-locations-config-and-local-sha

Reply:
yeah, could be an idea, I think we have to create the `.config` path if don't exist then, otherwise it would throw an error?

Reply:
Nope, you check for that Env, if it's not set then you're kind of free to use whatever you want. But it should be set in most distros.

Reply:
The spec
https://specifications.freedesktop.org/basedir-spec/latest/

So my suggestion is check if it exists otherwise use `$HOME/.pcomfortcloud-oauth-token`

Reply:
You should also avoid using the ~ .. I've seen issues with it. Try and use the home env

Reply:
Maybe `XDG_STATE_HOME` (defaults to `$HOME/.local/state/`) should be used instead of `XDG_CONFIG_HOME`.

From the specs cited above:
> The $XDG_STATE_HOME contains state data that should persist between (application) restarts, but that is not important or portable enough to the user that it should be stored in $XDG_DATA_HOME. It may contain:
> 
>  -   actions history (logs, history, recently used files, …)
> 
>  -   current state of the application that can be reused on a restart (view, layout, open files, undo history, …)

Reply:
`$HOME` doesn't work in Windows. Hmm.

Reply:
https://stackoverflow.com/questions/4028904/what-is-a-cross-platform-way-to-get-the-home-directory

```from pathlib import Path
home = str(Path.home())```